iosApp/Survey/Sources/Presentation/Modules/SurveyDetail/SurveyDetailView.swift

```diff
@@ -26,10 +27,21 @@ struct SurveyDetailView: View {
     // TODO: Replace with real answer object
     @State var currentAnswers = [String]()
     @State var isShowingQuitPrompt = false
+    @State var isShowingSuccess = false
```

Comment:
Optional: small naming suggestion for better clarity 🙏
```suggestion
    @State var isShowingSuccessConfirmation = false
```